pkg/services/notary/notary.go

```diff
@@ -272,7 +272,7 @@ func (n *Notary) OnNewRequest(payload *payload.P2PNotaryRequest) {
 		// Avoid changes in the main transaction witnesses got from the notary request pool to
 		// keep the pooled tx valid. We will update its copy => the copy's size will be changed.
 		r = &request{
-			main:              safeCopy(payload.MainTransaction),
+			main:              payload.MainTransaction.Copy(),
```

Comment:
> notary: refactor Copy() of transaction.Transaction 

Change the commit message to something more specific. It's not the `Copy()`'s method refactoring as it can be judged from the commit message. It's something like `notary: reuse (*Transaction).Copy where possible` or `notary: get rid of custom safeCopy implementation` or something like that.